content/en/real_user_monitoring/guide/proxy-mobile-rum-data.md

```diff
@@ -58,16 +58,32 @@ Datadog.initialize(
     env: "<environment>",
     proxyConfiguration: [
         kCFNetworkProxiesHTTPEnable: true,
-        kCFNetworkProxiesHTTPPort: 123,
-        kCFNetworkProxiesHTTPProxy: "www.example.com",
-        kCFProxyUsernameKey: "proxyuser",
-        kCFProxyPasswordKey: "proxypass"
+        kCFNetworkProxiesHTTPPort: <123>,
+        kCFNetworkProxiesHTTPProxy: "<www.example.com>",
+        kCFProxyUsernameKey: "<proxy user>",
+        kCFProxyPasswordKey: "<proxy password>"
     ]
   ),
   trackingConsent: trackingConsent
 )
 ```
 
+```objective-c
+@import DatadogObjc;
+
+DDConfiguration *configuration = [[DDConfiguration alloc] initWithClientToken:@"<client token>" env:@"<environment>"];
+configuration.proxyConfiguration = @{
+    (NSString *)kCFNetworkProxiesHTTPEnable: @YES,
+    (NSString *)kCFNetworkProxiesHTTPPort: @<123>,
+    (NSString *)kCFNetworkProxiesHTTPProxy: @"<www.example.com>",
+    (NSString *)kCFProxyUsernameKey: @"<proxyuser>",
+    (NSString *)kCFProxyPasswordKey: @"<proxypass>"
+}];
```

Comment:
Just wanted to flag that it looks like there's no opening bracket (`[`) for the closing bracket on line 81. Feel free to ignore this comment if I'm missing anything.

Reply:
Good catch it was a mistake I carried over from the previous doc, I will fix it!